server/repository/src/migrations/v2_08_00/add_vvm_status_log_update_to_activity_log.rs

```diff
@@ -0,0 +1,23 @@
+use crate::migrations::*;
+
+pub(crate) struct Migrate;
+
+impl MigrationFragment for Migrate {
+    fn identifier(&self) -> &'static str {
+        "add_vvm_status_log_update_to_activity_log"
+    }
+
+    fn migrate(&self, connection: &StorageConnection) -> anyhow::Result<()> {
+        if cfg!(feature = "postgres") {
+            sql!(
+                connection,
+                r#"
+                    ALTER TYPE activity_log_type
```

Comment:
Honestly... should be combined with [this](https://github.com/msupply-foundation/open-msupply/pull/7717/files#diff-28e38d97535f70357c8e4fa8476dca2a0da7ce432d80220abf9bf272204c7e29) 👀